bootstrapper-maven-plugin/src/main/resources/static/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.Authenticator;
+import java.net.PasswordAuthentication;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+public final class MavenWrapperDownloader
+{
+    private static final String WRAPPER_VERSION = "3.2.0";
```

Comment:
tracking and updating this version is gonna be challenging, sadly I don't have great alternatives 😢

Reply:
Why use a wrapper at all? That seems like it complicates things unduly, I mean it's pretty rare that things don't work because of maven these days.

Reply:
it is just for the case if some does not have maven or proper maven version, most of these generator (like in quarkus) these are generated. TBH no strong opinion on this, if both of you think we should rather remove it, I'm fine with that.
@metacosm @andreaTP

Reply:
I think it'd be simpler without it. Personally, I never use the wrapper unless I encounter a specific issue where I think maven maybe the cause…

Reply:
We can always re-add it later if people think it's better.